tests/model_registry/conftest.py

```diff
@@ -0,0 +1,315 @@
+        name=DB_RESOURCES_NAME,
+        namespace=model_registry_namespace.name,
+        ports=[
+            {
+                "name": "mysql",
+                "nodePort": 0,
+                "port": 3306,
+                "protocol": "TCP",
+                "appProtocol": "tcp",
+                "targetPort": 3306,
+            }
+        ],
+        selector={
+            "name": DB_RESOURCES_NAME,
+        },
+        label={
+            "app.kubernetes.io/name": DB_RESOURCES_NAME,
+            "app.kubernetes.io/instance": DB_RESOURCES_NAME,
+            "app.kubernetes.io/part-of": DB_RESOURCES_NAME,
+        },
+        annotations={
+            "template.openshift.io/expose-uri": "mysql://{.spec.clusterIP}:{.spec.ports[?(.name==\mysql\)].port}",
+        },
+    ) as mr_db_service:
+        yield mr_db_service
+
+
+@pytest.fixture(scope="class")
+def model_registry_db_pvc(
+    admin_client: DynamicClient,
+    model_registry_namespace: Namespace,
+) -> Generator[PersistentVolumeClaim, Any, Any]:
+    pvc_kwargs = {
+        "accessmodes": "ReadWriteOnce",
+        "name": DB_RESOURCES_NAME,
+        "namespace": model_registry_namespace.name,
+        "client": admin_client,
+        "size": "5Gi",
+        "label": {
+            "app.kubernetes.io/name": DB_RESOURCES_NAME,
+            "app.kubernetes.io/instance": DB_RESOURCES_NAME,
+            "app.kubernetes.io/part-of": DB_RESOURCES_NAME,
+        },
+    }
+
+    with PersistentVolumeClaim(**pvc_kwargs) as pvc:
+        yield pvc
+
+
+@pytest.fixture(scope="class")
+def model_registry_db_secret(
+    admin_client: DynamicClient,
+    model_registry_namespace: Namespace,
+) -> Generator[Secret, Any, Any]:
+    with Secret(
+        client=admin_client,
+        name=DB_RESOURCES_NAME,
+        namespace=model_registry_namespace.name,
+        string_data={
+            "database-name": "model_registry",
+            "database-password": "TheBlurstOfTimes",  # pragma: allowlist secret
+            "database-user": "mlmduser",  # pragma: allowlist secret
+        },
+        label={
+            "app.kubernetes.io/name": DB_RESOURCES_NAME,
+            "app.kubernetes.io/instance": DB_RESOURCES_NAME,
+            "app.kubernetes.io/part-of": DB_RESOURCES_NAME,
+        },
+        annotations={
+            "template.openshift.io/expose-database_name": "'{.data[''database-name'']}'",
+            "template.openshift.io/expose-password": "'{.data[''database-password'']}'",
+            "template.openshift.io/expose-username": "'{.data[''database-user'']}'",
+        },
+    ) as mr_db_secret:
+        yield mr_db_secret
+
+
+@pytest.fixture(scope="class")
+def model_registry_db_deployment(
+    admin_client: DynamicClient,
+    model_registry_namespace: Namespace,
+    model_registry_db_secret: Secret,
+    model_registry_db_pvc: PersistentVolumeClaim,
+    model_registry_db_service: Service,
+) -> Generator[Deployment, Any, Any]:
+    with Deployment(
```

Comment:
is there a template for this deployment in the cluster?

Reply:
No